src/ctype/src/lib.rs

```diff
@@ -80,18 +80,10 @@ pub extern "C" fn toascii(c: c_int) -> c_int {
 
 #[no_mangle]
 pub extern "C" fn tolower(c: c_int) -> c_int {
-    if isupper(c) != 0 {
-        c + 0x20
-    } else {
-        c
-    }
+    if isupper(c) != 0 { c + 0x20 } else { c }
```

Comment:
Our rustfmt.toml shouldn't allow this.

Reply:
I ran cargo fmt instead of ./fmt.sh by accident

Reply:
Ah np